prisma/migrations/20210707085343_releases/migration.sql

```diff
@@ -11,3 +11,7 @@ CREATE TABLE "release" (
 
 -- CreateIndex
 CREATE INDEX "release.release_time_index" ON "release"("release_time");
+
+INSERT INTO release (id, release_name, release_time, release_tag)
+VALUES (1,'do auto migrate in start','2021-07-02T13:29:57Z','0.0.4') ON CONFLICT DO NOTHING;
+
```

Comment:
A migration should not add content to tables. We should use [Prisma's seed functionality](https://www.prisma.io/docs/guides/database/seed-database#requirements-for-seeding-with-typescript-or-javascript) for that.